docs/dev/building-on-zksync/interacting.md

```diff
@@ -55,7 +55,7 @@ In order to add the zkSync Era alpha mainnet network to your wallet, you will ne
 - RPC URL: `https://testnet.era.zksync.dev`
 - Chain ID: `280`
 - Currency Symbol: `ETH`
-- Block Explorer URL: `https://goerli.explorer.zksync.io/`
+- Block Explorer URL: `https://sepolia.explorer.zksync.io/` or `https://goerli.explorer.zksync.io/`
 - WebSocket URL: `wss://testnet.era.zksync.dev/ws`
```

Comment:
This needs to be updated. Please confirm the new chainID and RPC URL. The block explorer URL here should reflect the testnet in which users will input into MM so this field cannot be one or the other.

(1) Goerli (2) Sepolia. We should list Sepolia first, and Goerli second until we fully sunset. We can add a ! indicating Goerli will be deprecated in the new year.

Reply:
Correct. I've already collected the right information. The changes will come in the upcoming commits.

- Great, will keep Sepolia first and Goerli second as the order in other instances, thank you.